### DISCUSSION TITLE
Removing TextEdit padding

### DISCUSSION THREAD

**Discussion by TimUntersberger** [emilk/egui#769] [Q&A]
Hello,

how can I remove the padding inside a textedit? I looked at the source code and found a hardcoded margin variable in the ui method of TextEdit, so it might not be possible yet. I just wanted to make sure I am not missing anything.

**Comment by emilk** [comment]
Yeah, that margin is currently hard-coded. It should probably be moved to to `Spacing` so it can be easily changed globally and locally.
